### PR TITLE
Fixes VM service breakage due to deprecated method

### PR DESCRIPTION
**Pull request by elliette**
Fixes error:

```
lib/src/vm_service_wrapper.dart:853:20 - 'setExceptionPauseMode' is deprecated and shouldn't be used. Use setIsolatePauseMode instead. Try replacing the use of the deprecated member with the replacement. - deprecated_member_use
```

### CODE REVIEW

File: packages/devtools_app/lib/src/vm_service_wrapper.dart [hunk 1]
```diff
@@ -841,9 +841,9 @@ class VmServiceWrapper implements VmService {
   }
 
   @override
-  Future<Success> setExceptionPauseMode(String isolateId, String mode) {
-    return trackFuture('setExceptionPauseMode',
-        _vmService.setExceptionPauseMode(isolateId, mode));
+  Future<Success> setIsolatePauseMode(String isolateId, String mode) {
```
Comment (kenzieschmoll) [inline packages/devtools_app/lib/src/vm_service_wrapper.dart:844]: based on the bot failures, looks like the method signature needs to be changed to `Future<Success> Function(String, {String exceptionPauseMode, bool shouldPauseOnExit})`

File: packages/devtools_app/lib/src/debugger/debugger_controller.dart [hunk 2]
```diff
@@ -538,7 +538,7 @@ class DebuggerController extends DisposableController
   }
 
   Future<void> setExceptionPauseMode(String mode) async {
```
Comment (kenzieschmoll) [inline packages/devtools_app/lib/src/debugger/debugger_controller.dart:540]: rename this method to `setIsolatePauseMode`

Comment (elliette) [reply packages/devtools_app/lib/src/debugger/debugger_controller.dart]: Done!

File: packages/devtools_app/lib/src/vm_service_wrapper.dart [hunk 3]
```diff
@@ -841,9 +841,19 @@ class VmServiceWrapper implements VmService {
   }
 
   @override
-  Future<Success> setExceptionPauseMode(String isolateId, String mode) {
-    return trackFuture('setExceptionPauseMode',
-        _vmService.setExceptionPauseMode(isolateId, mode));
+  Future<Success> setIsolatePauseMode(
+    String isolateId, {
+    /*ExceptionPauseMode*/ String exceptionPauseMode,
```
Comment (kenzieschmoll) [inline packages/devtools_app/lib/src/vm_service_wrapper.dart:846]: why is this commented out like this?

Comment (kenzieschmoll) [reply packages/devtools_app/lib/src/vm_service_wrapper.dart:846]: sg

**Review by kenzieschmoll — APPROVED** [review]

**Pull request merged**